### ISSUE TITLE
citation key as template string

### ISSUE THREAD

**Issue by jrabensc** [wbthomason/zotodo#3]
Absolutely great plugin, I was looking for something like this for years now.

Here's a small request: It would be really helpful when we could add the citation key as a template string into the Todoist title or comment.

**Comment by wbthomason** [comment]
Yeah, that's something I've meant to add in for myself but haven't gotten to yet. Shouldn't be too hard.

**Comment by wbthomason** [comment]
As a small update, I'm trying to figure out how to get this field from items. It only exists if you have better-bibtex/similar installed, so the documentation for its access is nonexistent (or at least I haven't found it yet).

**Comment by argenos** [comment]
I recently used this on a plugin of my own, and based it on [this](https://github.com/melat0nin/zotero-roam-export/blob/master/chrome/content/roamexport.js#L47):

```
   getBBTCiteKey(item) {
        if (typeof Zotero.BetterBibTeX === 'object' && Zotero.BetterBibTeX !== null) {
            var bbtItem = Zotero.BetterBibTeX.KeyManager.get(item.getField('id')), 
                bbtCiteKey = bbtItem.citekey;
            return bbtCiteKey;
        }
        return false;
    }
```

**Comment by wbthomason** [comment]
Ah, fantastic! Thank you; I'll incorporate this now.

**Comment by wbthomason** [comment]
Thanks again, @argenos! The template string key `citekey` is now included in v0.6.0.
